externals/kyuubi-flink-sql-engine/src/test/scala/org/apache/kyuubi/engine/flink/result/ResultSetSuite.scala

```diff
@@ -44,9 +46,9 @@ class ResultSetSuite extends KyuubiFunSuite {
       .data(rowsNew)
       .build
 
-    assert(RowSet.toRowBaseSet(rowsNew, resultSetNew)
-      === RowSet.toRowBaseSet(rowsOld, resultSetOld))
-    assert(RowSet.toColumnBasedSet(rowsNew, resultSetNew)
-      === RowSet.toColumnBasedSet(rowsOld, resultSetOld))
+    assert(RowSet.toRowBaseSet(rowsNew, resultSetNew, ZoneId.systemDefault())
```

Comment:
`ZoneId.systemDefault()` may cause different results in different environments, let's use a specific one, e.g. `America/Los_Angeles`(PST) `Asia/Shanghai`(CST)